### PR TITLE
ODC-7600: Add Timeout field in the start pipeline modal

### PR DESCRIPTION
**Pull request by vikram-raj**
Add a Timeout field in the start pipeline modal

https://github.com/openshift/console/assets/2561818/336610c9-ec87-44d6-8c18-c31a8e12ab94

### CODE REVIEW

**Review by lokanandaprabhu — CHANGES_REQUESTED** [review]
If I don't enter any value in timeout I am getting error message

Screenshot 2024-06-12 at 12 10 30 PM

File: src/components/start-pipeline/utils.ts [hunk 1]
```diff
@@ -331,8 +331,8 @@ export const getPipelineRunFromForm = (
   annotations?: { [key: string]: string },
   options?: { generateName: boolean },
 ) => {
-  const { parameters, workspaces } = formValues;
-
+  const { parameters, workspaces, timeouts } = formValues;
+  console.log(timeouts, '####');
```
Comment (lokanandaprabhu) [inline src/components/start-pipeline/utils.ts:335]: Can you please remove this log

**Comment by lokanandaprabhu** [comment]
Due to new form group, the space is more above Timeouts section, can we add this as part of Parameter section or remove the additional space? 

Screenshot 2024-06-12 at 12 32 00 PM

**Comment by vikram-raj** [comment]
> Due to new form group, the space is more above Timeouts section, can we add this as part of Parameter section or remove the additional space?

I added it in a different section as it is not a part of the parameters. Timeout is a different section.

**Review by lokanandaprabhu — APPROVED** [review]

**Pull request merged**